index.html

```diff
@@ -3,184 +3,196 @@
+                <!-- TODO: Make the thing below a carousel with older group pictures -->
+            <img class="dual_container_item" src="assets/images/welcome_back_large.webp" alt="CSH stands together on a grass field"/>
+        </div>                
+    </div>                
+</div>
+
+<!-- Projects tab -->
+<h3 class="left_offset" style="padding-top: 0.6em;">Things we're working on ...</h3>
+<div class="quad_image">
+    <div class="content">
+        <a href="csh.rit.edu">
+            <div class="content-overlay"></div>
+            <img class="content-image" src="assets/images/server.png" style="width: 100%;" alt="A picture of CSH's server racks"/>
+            <div class="content-details fadeIn-bottom"">
+                <h3 class="content-title">Server room upgrade</h3>
+                <p>Done in collaboration with wayfair</p>
+            </div>
+        </a>
+    </div>
+    <div class="content">
+        <a href="csh.rit.edu">
+            <div class="content-overlay"></div>
+            <img class="content-image" src="assets/images/TUNES.png" style="width: 100%;" alt="TUNES, an old jukebox that's being made digital"/>
+            <div class="content-details fadeIn-bottom">
+                <h3 class="content-title">TUNES</h3>
+                <p>Modernizing a jukebox</p>
+            </div>
+        </a>
+    </div>
+    <div class="content">
+        <div class="content-overlay"></div>
+        <img class="content-image" src="assets/images/infosys.png" style="width: 100%;" alt="A LED matrix that makes up CSH's Infosys"/>
+        <div class="content-details fadeIn-bottom">
+            <h3 class="content-title">Infosys</h3>
+            <p>A legacy CSH project that is being modernized.</p>
         </div>
-        {% for post in site.posts limit:2 %}
-            {% include post.html post=post %}
-        {% endfor %}
-        <div class="col-12">
-            <a class="btn btn-primary" href="/blog/">More blog posts</a>
+    </div>
+    <div class="content">
+        <div class="content-overlay"></div>
+        <img class="content-image" src="assets/images/letmein.png" style="width: 100%;" alt="A raspberry pi and pencil sharpener on the wall, CSH's letmein system"/>
+        <div class="content-details fadeIn-bottom">
+            <h3 class="content-title">Letmein</h3>
+            <p>A legacy CSH project that is being modernized.</p>
         </div>
     </div>
 </div>
 
-<div class="gray-wrapper">
-    <div class="container spaced" id="homepage-projects">
-        <div class="row">
-            <div class="col-12">
-                <h2 class="header">Things we've done</h2>
+<div  style="padding-top: 0.5em; padding-bottom: 0.5em; margin-bottom: 0%;" id="stats">
+    <h3 class="left_offset" style="padding-top: 0.6em; margin-top: 0em;">Some facts about us ...</h3>
+    <div class="width_capped">
+        
+        <div class="container_stat">
+            <div class="stat_card">
+                <p class="stat_big">315</p>
```

Comment:
What's this number based on?

Reply:
It was a placeholder that I forgot about. Fixed